packages/webpack/get-config.js

```diff
@@ -1,27 +1,14 @@
 let StatoscopeWebpackPlugin = require('@statoscope/webpack-plugin').default
-let { writeFile } = require('fs').promises
+let updateCheckWithImports = require('size-limit/updateCheckWithImports')
```

Comment:
I do not use camel case in other files in the project. Be careful.

Reply:
Thanks for the point! I will keep an eye on it